InventoryManagementSystem_SQL/Model/ProductQueries.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace InventoryManagementSystem_SQL.Model
+{
+  
+        public static class ProductQueries
+        {
+            public static string GetAllProducts()
+            {
+                return "SELECT * FROM Products";
+            }
+
+            public static string IsEmpty()
+            {
+                return "SELECT TOP 1 * FROM Products";
```

Comment:
you can re-query this as "SELECT count(*) FROM Products" and check then if result =0